source/auth/auth.rst

```diff
@@ -610,16 +610,10 @@ follows:
 
 This same conversation over mongodb's sasl implementation would appear as follows:
 
-| C: :javascript:`{saslStart: 1, mechanism: "SCRAM-SHA-1", payload: BinData(0, "biwsbj11c2VyLHI9ZnlrbytkMmxiYkZnT05Sdjlxa3hkYXdM")}`
+| C: :javascript:`{saslStart: 1, mechanism: "SCRAM-SHA-1", payload: BinData(0, "biwsbj11c2VyLHI9ZnlrbytkMmxiYkZnT05Sdjlxa3hkYXdM"), options: { skipEmptyExchange: true }}`
```

Comment:
Let's copy this section (with appropriate changes to the payloads) to the SCRAM-SHA-256 section, so it's clear that drivers should implement skipEmptyExchange for that mechanism too.

Reply:
Done